### ISSUE TITLE
Finished with 0

### ISSUE THREAD

**Issue by shaunk0111** [winsw/winsw#231]
I'm running an application in CLI mode, running on a windows 7 machine. I can install the application as a service with 2.1.0 with no errors. I can also start the service successfully, although the application is exiting unexpectedly after starting. 

**Install**
-> app.exe install
2017-07-06 09:37:04,265 INFO  - Starting ServiceWrapper in the CLI mode
2017-07-06 09:37:04,481 INFO  - Completed. Exit code is 0

**This is the Event Occurring**
Child process [5804 - C:\app\app.exe C:\app\st.cmd] finished with 0.

**wrapper.log**
2017-07-05 15:42:35,547 INFO  - Starting ServiceWrapper in the service mode
2017-07-05 15:42:35,564 INFO  - Starting C:\app\app.exe 
2017-07-05 15:42:35,567 INFO  - Starting C:\app\app.exe 
2017-07-05 15:42:35,575 INFO  - Started process 5048

**err.log**
Starting iocInit
iocRun: All initialization complete

These lines in the error log are not errors, which is strange to me. 

To summarize, the application appears to be starting correctly, it calls a few different applications through start up, then opens a console interface (no window pops up), then exits. I see its working correctly through the out.log. Is there any reason this could be occurring?

**app.xml**
```
 <service>
      <id>app</id>
      <name>App</name>
      <description>My App</description>
      <executable>C:\app\app.exe</executable>
      <arguments>C:\app\st.cmd</arguments>
      <logmode>rotate</logmode>
 </service>
```

**Comment by oleg-nenashev** [comment]
It duplicates #198

**Comment by shaunk0111** [comment]
I there a work around for this? #198 _'Fixing the folder permissions gets rid of this specific problem'_. My folder permission have full control for all users. Do i need to install the services as specific user or service type to resolve this?

**Comment by oleg-nenashev** [comment]
@shaunk0111 Well, I was rather referring to the zero return code which causes poor issue diagnostics. 

Some console applications may fail to start as services if they actually include UI components/options. Just to ensure it is not a cause, please try running with after installing service with the `interactive` flag

**Comment by NextTurn** [comment]
What exactly did "opens a console interface" do? Did it invoke `cmd.exe /k`? Did it read from STDIN to block?

#610 probably fixed this.
